data/ramanand-rv.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Ramanand Thakur",
+  "type": "personal",
+  "bio": "A Software Developer and AI/ML enthusiast. Code👨🏻‍💻->Coffee🍵->Commit🔀->Repeat🔁",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn🤝🏻",
+      "url": "https://www.linkedin.com/in/ramanand-rv",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub🐼",
+      "url": "https://www.github.com/ramanand-rv/",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter🐤",
+      "url": "https://twitter.com/ramanand_rv",
+      "icon": "FaTwitter"
+    },
+
+    {
+      "name": "YouTube▶️",
+      "url": "https://www.youtube.com/@ramanandthakur1464",
+      "icon": "FaYouTube"
```

Comment:
Hi @ramanand-rv, thank you for contributing  :tada: 
Please replace it with **' FaYoutube '** :+1: 
```Suggestion
      "icon": "FaYoutube"
```

Reply:
Thank you @khushalarora, I've the requested change.